### PR TITLE
Add 'view data' functionality to the web ui

### PR DESCRIPTION
**Pull request by lkuchenb**
Adds a table of all submitted metadata to the "view data" page. Extraction of (raw) data is not possible at this stage.

### CODE REVIEW

**Comment by lkuchenb** [comment]
Sorting still WIP

**Comment by lkuchenb** [comment]
View is now functional provides case-insensitive searching and lexicographic column ordering.

File: datameta/api/ui/view.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+        length   = int(request.POST['length'])
+        # The first record to return, 0 based
+        start    = int(request.POST['start'])
+        # A search pattern the user entered in the search field of the table,
+        # may be empty
+        search   = request.POST['search[value]']
+        # The column index for sorting
+        sort_idx = int(request.POST['order[0][column]'])
+        # The sorting direction
+        sort_asc = request.POST['order[0][dir]'] == 'asc'
```
Comment (cschu) [inline datameta/api/ui/view.py:83]: is this caught somewhere that the values can only be `asc` and `desc` or will `anything` also do descending sort?

Comment (lkuchenb) [reply datameta/api/ui/view.py:83]: No, that's not caught. Kept it simple here, since we're talking to an API on the other end.

**Pull request merged**